src/Persistence/Models/Control.cs

```diff
@@ -56,11 +61,55 @@ public Control(string name, ControlTemplate template)
     /// list of child controls nested under this control.
     /// </summary>    
     [YamlMember(Order = 3)]
-    public Control[] Children { get; init; } = Array.Empty<Control>();
+    [SuppressMessage("Style", "IDE0032:Use auto property", Justification = "We need both 'public init' and 'private set', which cannot be accomplished by auto property")]
+    public Control[] Children { get => _children; init => _children = value; }
 
     [YamlIgnore]
     public ControlEditorState? EditorState { get; set; }
 
     [YamlIgnore]
     public required ControlTemplate Template { get; init; }
+
+    [OnDeserialized]
+    internal void PostDeserialize()
```

Comment:
nit: use `Before` and `After`

Reply:
done in https://github.com/microsoft/PowerApps-Tooling/pull/580